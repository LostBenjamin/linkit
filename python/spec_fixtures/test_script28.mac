
$asdf  1

[ testing ]
  ( test2 )
  ( test2 < asdf > )

[ test2 ]
  stp

