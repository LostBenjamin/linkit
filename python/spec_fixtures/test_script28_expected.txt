4:set:stp
5:set:4:run:1,4:sch

